crates/egui/src/memory.rs

```diff
@@ -487,6 +487,11 @@ impl Memory {
     pub fn reset_areas(&mut self) {
         self.areas = Default::default();
     }
+
+    /// Obtain the previous area rect of a window.
+    pub fn window_rect(&self, id: impl Into<Id>) -> Option<Rect> {
```

Comment:
It would be great to link to this function from the docs of `Window` and `Area`

Reply:
Done!